rs/execution_environment/src/canister_manager.rs

```diff
@@ -1797,7 +1797,11 @@ impl CanisterManager {
         system_state
             .certified_data
             .clone_from(snapshot.certified_data());
-
+        // TODO: EXC-2004 let the user decide whether to load these or not
+        // system_state.global_timer = snapshot.execution_snapshot().global_timer;
+        // system_state
+        //     .task_queue
+        //     .set_hook_status(snapshot.execution_snapshot().on_low_wasm_memory_hook_status);
```

Comment:
This seems to be unnecessary. I would just live it out and then add it when we actually implement the functionality.